sessions/forecasting-models.qmd

```diff
@@ -312,7 +324,19 @@ target_onsets <- onset_df |>
   filter(day <= cutoff + horizon)
 ```
 
+
 ```{r plot_forecast}
+forecast |>
+  filter(.draw %in% sample(.draw, 100)) |>
+  ggplot(aes(x = day)) +
+  geom_line(alpha = 0.1, aes(y = .value, group = interaction(.draw, model), colour = model)) +
+  geom_point(data = target_onsets, aes(x = day, y = onsets), color = "black") +
+  guides(colour = guide_legend(override.aes = list(alpha = 1)))
```

Comment:
```suggestion
  guides(colour = guide_legend(override.aes = list(alpha = 1))) +
  lims(y = c(0, 500))
```

Suggest adding upper limit to make the plot readable (same limit as in the later plot of all forecasts)